resources/lang/de.json

```
@@ -634,12 +634,39 @@
     "menu.oauth_authorize.authorize": "Autorisieren",
     "menu.oauth_authorize.cancel": "Abbrechen",
     "menu.oauth_authorize.webhook_request": "Diese Anwendung möchte über die folgenden Aktivitäten auf deinem Träwelling Account benachrichtigt werden:",
-    "menu.oauth_authorize.request": "<span class=\"fw-bold\">:application</span> is requesting permission to access your account.",
+    "menu.oauth_authorize.request": "<span class=\"fw-bold\">:application</span> bittet um Erlaubnis, auf dein Konto zuzugreifen.",
+    "menu.oauth_authorize.scopes_title": "Diese Anwendung wird folgendes tun können:",
```

"tun" würde ich lieber weglassen - das ist so eine doofe Umgangssprache. :D 

```suggestion
    "menu.oauth_authorize.scopes_title": "Diese Anwendung möchte folgende Berechtigungen:",
```